hermes-common/src/main/java/pl/allegro/tech/hermes/common/metric/counter/zookeeper/ZookeeperCounterReporter.java

```diff
@@ -93,7 +93,7 @@ private static TopicName escapedTopicName(String qualifiedTopicName) {
         TopicName topicName = fromQualifiedName(qualifiedTopicName);
         return new TopicName(
                 escapeMetricsReplacementChar(topicName.getGroupName()),
-                escapeMetricsReplacementChar(topicName.getName())
+                topicName.getName()
```

Comment:
Currently we don't have underscores in groups names on prod and in this PR I am adding restriction to proper field in hermes-console, so this situation will not occur in the future. 

When metric is created in MetricRegistry then Hermes escapes group, topic and subscription names from `.` to `_`. However topic name can't have dots so this transformation will do nothing for topic name. This is the reason why we can use underscore in topic name but not in group and subscription name. In ZookeeperCounterReporter Hermes again transforms `_` to `.` to keep original names but it shouldn't do this for topic name.